### PR TITLE
update to get grpc js

### PR DESCRIPTION
**Pull request by f1nn3g4n**
hoping that this is sufficient to get the ng files to generate with grpc js on the front end!

### CODE REVIEW

**Comment by f1nn3g4n** [comment]
Needs to be combined with work in the Minga repo to test

File: WORKSPACE [hunk 1]
```diff
@@ -3,8 +3,27 @@ workspace(name = "com_github_zaucy_protoc_gen_angular")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
-  name = "com_google_protobuf",
+  name = "com_google_protobuf_old",
```
Comment (ken31ee) [inline WORKSPACE]: not sure if this will work. need to test with the gateway

Comment (f1nn3g4n) [reply WORKSPACE]: I switched to the gateway version of protobuf, should it work now @ken31ee ?

**Review by mattchangegamers — APPROVED** [review]

File: WORKSPACE (hunk 1 above)
Comment (ken31ee) [reply WORKSPACE]: Theoretically yes. The app and gateway should be able to comunicate without problem

**Pull request merged**